include/solvers/vof_filter.h

```diff
@@ -21,46 +21,61 @@
 #include <core/parameters_multiphysics.h>
 
 /**
- * @brief Class that allows phase fraction filtering
+ * @brief Abstract class for filtering phase fraction in volume of fluid (VOF)
+ * simulations.
```

Comment:
I would remove the abstract class. What do you think?

Reply:
I think that the **brief** should say that you can't directly instantiate an object of the class; should it not?

Reply:
Agreed